go.mod

```diff
@@ -10,5 +10,5 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/surma/gocpio v1.1.0
 	github.com/ulikunitz/xz v0.5.12
-	golang.org/x/sys v0.22.0
+	golang.org/x/sys v0.24.0
```

Comment:
Lets wait till 0.24 actually hits at least debian unstable or ideally 0.26 as that was more recently released; Arguably we should have a CI tests for "does this compile in testing/unstable" :)

Reply:
Good idea; a separate job to install the Debian build deps (e.g go packages) and build with go from Debian.
I'll look at adding this job for both debos/fakemachine.